images/jdk/configs/latest-dev.apko.yaml

```diff
@@ -0,0 +1,39 @@
+contents:
+  repositories:
+    - https://packages.wolfi.dev/os
+  keyring:
+    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
+  packages:
+    - ca-certificates-bundle
+    - wolfi-baselayout
+    - glibc-locale-en
+    - busybox
+    - bash
+    - openjdk-17
+
+accounts:
+  groups:
+    - groupname: java
+      gid: 65532
+  users:
+    - username: java
+      uid: 65532
+  run-as: java
+  recursive: true
+
+work-dir: /home/build
+
+environment:
+  LANG: en_US.UTF-8
+  JAVA_HOME: /usr/lib/jvm/openjdk
+
+paths:
+  - path: /home/build
+    type: directory
+    uid: 65532
+    gid: 65532
+    permissions: 0o755
```

Comment:
is using octal format the preferred way here?

Reply:
yes? https://github.com/chainguard-dev/apko/blob/main/examples/nginx.yaml#L32